tests/console/ansible.pm

```diff
@@ -0,0 +1,125 @@
+
+    # Install ansible
+    #zypper_call 'in ansible jq ansible-test';
+
+    # Instal Ansible from PIPY
+    assert_script_run 'pip3 install ansible', 300;
+    #assert_script_run 'export PATH=/home/bernhard/.local/bin:$PATH';
+
+    # Enable sshd
+    systemctl 'start sshd';
+
+    # add $testapi::username to sudoers without password
+    assert_script_run "echo '$testapi::username ALL=(ALL:ALL) NOPASSWD: ALL' | tee -a /etc/sudoers";
+    #assert_script_run "echo 'root ALL=(ALL:ALL) NOPASSWD: ALL' | tee -a /etc/sudoers";
+
+    # Logout root and login $testapi::username
+    enter_cmd 'exit';
+    $self->select_user_serial_terminal;
+
+    # Check that we are logged in as $testapi::username
+    validate_script_output('whoami', sub { m/$testapi::username/ });
+
+    # Check that we have sudo root permissions
+    validate_script_output('sudo whoami', sub { m/root/ });
+
+    # Generate RSA key if needed
+    assert_script_run 'test ! -f ~/.ssh/id_rsa.pub && ssh-keygen -t rsa -P "" -C "$USER@localhost" -f ~/.ssh/id_rsa';
+
+    # Make sure our public key is un the authorized_keys file
+    assert_script_run 'cat ~/.ssh/id_rsa.pub | tee -a ~/.ssh/authorized_keys';
+
+    # Learn public SSH host keys
+    assert_script_run 'ssh-keyscan localhost >> ~/.ssh/known_hosts';
+
+    # Check that we can connect to localhost via SSH
+    validate_script_output('ssh localhost whoami', sub { m/$testapi::username/ });
+
+    my $hostname = script_output 'hostname';
+    validate_script_output 'ansible -m setup localhost | grep ansible_hostname', sub { m/$hostname/ };
+
+    my $arch = get_var 'ARCH';
+    validate_script_output 'ansible -m setup localhost | grep ansible_architecture', sub { m/$arch/ };
+
+    # Download data/console/ansible/ directory
+    assert_script_run 'curl ' . data_url('console/ansible/') . ' | cpio -id';
+
+    # In order to use ansible-test we need to follow this directory tree 
+    assert_script_run "mkdir -p ~/ansible_collections/openqa";
+    assert_script_run "mv data ~/ansible_collections/openqa/ansible";
+    assert_script_run 'cd ~/ansible_collections/openqa/ansible';
+
+    # Download base folder files
+    #assert_script_run 'curl -so main.yaml ' . data_url 'console/ansible/playbook_main.yaml';
+    #assert_script_run 'curl -so hosts ' . data_url 'console/ansible/playbook_hosts';
+
+    # Download role specific files
+    #assert_script_run 'curl -so roles/test/tasks/main.yaml ' . data_url 'console/ansible/role_task.yaml';
+    #assert_script_run 'curl -so roles/test/files/static.txt ' . data_url 'console/ansible/role_file.txt';
+    #assert_script_run 'curl -so roles/test/handlers/main.yaml ' . data_url 'console/ansible/role_handler.yaml';
+    #assert_script_run 'curl -so roles/test/templates/README.txt.j2 ' . data_url 'console/ansible/role_template.txt.j2';
+
+    # Download community.general modules because of zypper module
+    assert_script_run 'ansible-galaxy collection install community.general';
```

Comment:
Is this necessary? Isn't the `zypper` module available by default?

Reply:
It's not necessary.